lib/BridgedRoom.js

```diff
@@ -344,12 +344,11 @@ BridgedRoom.prototype._handleSlackMessage = function(message, ghost) {
                 let thumbnail_promise = Promise.resolve();
                 // Slack ain't a believer in consistency.
                 const thumb_uri = file.thumb_video || file.thumb_360;
-                if (thumb_uri) {
+                if (thumb_uri && file.filetype) {
                     thumbnail_promise = ghost.uploadContentFromURI(
                         {
-                            // Yes, we hardcode jpeg. Slack always use em.
-                            title: `${file.name}_thumb.jpeg`,
-                            mimetype: "image/jpeg",
+                            title: `${file.name}_thumb.${file.filetype}`,
+                            mimetype: "image/" + file.filetype,
```

Comment:
```suggestion
                            mimetype: file.mimetype,
```

`file.mimetype` would probably lead to more predictable and correct results here. (This is the way I implemented this change on our instance of the bridge at hacklab.fi)

Reply:
@vurpo method we also tested and confirmed working with multiple diffirent image formats on said hacklab.fi bridge.

Reply:
Makes sense. Cheers!